src/flag_gems/runtime/error/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .error import *  # noqa: F403
+
+__all__ = ["*"]
```

Comment:
If the module is simple, do not wrap it into a package. A simple module is enough.